### PR TITLE
JSCS for send-mail.js

### PR DESCRIPTION
**Pull request by Gomez**
One more file.
209 errors to go

### CODE REVIEW

**Comment by Gomez** [comment]
@DeepDiver1975 @wurstchristoph @clementhk @Xenopathic Easy review..

File: js/send-mail.js [hunk 1]
```diff
@@ -151,14 +151,16 @@ $(function () {
...
+		newMessageCcBcc.find('#cc').focus();
 		$('#new-message-cc-bcc-toggle').fadeOut();
 	});
 
 	$(document).on('click', '#reply-message-cc-bcc-toggle', function() {
-		$('#reply-message-cc-bcc').slideToggle();
-		$('#reply-message-cc-bcc #cc').focus();
+		var replyMessageCcBcc = $('#reply-message-cc-bcc');
+		replyMessageCcBcc.slideToggle();
+		replyMessageCcBcc.find('#cc').focus();
```
Comment (wurstchristoph) [inline js/send-mail.js]: Is this replacement really the same? I may be wrong, but to me it looks like it was a selection if any element with classes ``reply-message-cc-bcc`` and ``cc`` before.
Your changes select any element of class ``reply-message-cc-bcc``, but then ``find`` searches any descendants of that selection, see https://api.jquery.com/find/

Comment (Gomez) [inline js/send-mail.js]: Possible, i will check it in a other pr. Reverted it here, as it has nothing todo with jscs. THX

**Comment by wurstchristoph** [comment]
:+1:

**Pull request merged**